server/server.go

```diff
@@ -0,0 +1,123 @@
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/gorilla/mux"
+	"net/http"
+
+	"github.com/Clever/cron-admin/db"
+)
+
+func byteHandler(handler func(*http.Request) ([]byte, error)) func(http.ResponseWriter, *http.Request) {
+	return func(rw http.ResponseWriter, req *http.Request) {
+		data, err := handler(req)
+		if err != nil {
+			fmt.Printf("Received handler err: %s\n", err)
```

Comment:
I don't recall the exact details, but I believe at the time, Hall Monitor didn't have any 400s so it was safe to just call any error a 500. It seems like either the pattern could be updated to include a different type of error for 400s or another approach could be used.